ietf-network-slice-service.yang

```diff
@@ -0,0 +1,1112 @@
+module ietf-network-slice-service {

```

Comment:
Same here - it's not easy to generate the delta changes you are introducing - can you please fix it?